median_threshold: 12
boundX: 0.012
boundY: 0.012
boundÎ–: 0.012
